ethclient/ethclient.go

```diff
@@ -258,6 +258,19 @@ func (ec *Client) SubscribeNewHead(ctx context.Context, ch chan<- *types.Header)
 
 // State Access
 
+// NetworkID returns the network ID (also known as the chain ID) for this chain.
+func (ec *Client) NetworkID(ctx context.Context) (version *big.Int, err error) {
+	version = big.NewInt(0)
+
+	var ver string
+	if err = ec.c.CallContext(ctx, &ver, "net_version"); err != nil {
+		return version, err
+	}
+
+	version.SetString(ver, 10)
+	return
+}
```

Comment:
Please remove return value names.
Please parse into version directly instead of doing the parsing via SetString.

```go
version := new(big.Int)
err := ec.c.CallContext(ctx, &version, "net_version")
return version, err
```